pkg/component/worker/utils_windows.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2023 k0s authors
+
+Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package worker
+
+import (
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/Microsoft/hcsshim"
+	"github.com/avast/retry-go"
+	"github.com/sirupsen/logrus"
+)
+
+// PowerShell struct
+type PowerShell struct {
+	powerShell string
+}
+
+// New create new session
+func NewPowershell() *PowerShell {
+	ps, _ := exec.LookPath("powershell.exe")
```

Comment:
Would be nice to add error handling here. Also we should take a note on PowerShell in the external runtime dependencies section of the docs. Windows still simply states "TBD" :-)